docs/api/logger.rst

```diff
@@ -1,6 +1,23 @@
 Logger module
 =============
 
+Each module of :program:`rebase-helper` uses a dedicated logger. Its name corresponds to
+the position in the module hierarchy (e.g. module :mod:`rebasehelper.application` uses the logger `rebasehelper.application`).
+The :class:`rebasehelper.logger.CustomLogger` class is used instead of `logging.Logger` to add more levels of logging.
+
+By default, all loggers are set to logging level `DEBUG` to ensure that all messages are caught.
+No handlers are created for the loggers, :program:`rebase-helper` adds them in :meth:`rebasehelper.cli.CliHelper.run`.
```

Comment:
Also unimportant. This is implementation detail, instead you should describe how to set up default handles with `rebasehelper.logger.create_stream_handlers()`.